src/modules/Microsoft.FinOpsHubs/datasets/deploy.bicep

```diff
@@ -0,0 +1,91 @@
+// Source: 
+// Date: 2023-02-02
+// Version: 
+
+@description('Conditional. The name of the parent Azure Data Factory. Required if the template is used in a standalone deployment.')
+param dataFactoryName string
+
+@description('Required. The name of the dataset.')
+param datasetName string
+
+@description('Required. The name of the dataset linked service.')
+param linkedServiceName string
+
+@description('Required. The type of dataset.')
+param datasetType string
+
+@description('Required. The type of dataset.')
+param locationType string = 'AzureBlobFSLocation'
+
+@description('Optional. The type of dataset.')
+param compressionCodec string = 'none'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
```

Comment:
If this is a custom template, there's no need for us to reuse the defaultTelemetry.

Reply:
Done.